### DISCUSSION TITLE
Support ZTE F50 5G modem

### DISCUSSION THREAD

**Discussion by peterwillcn** [immortalwrt/immortalwrt#1629] [General]
### Describe the bug

Support [ZTE F50 5G](https://www.ztedevices.com/cn/product/zte-f50-5g-ufi/) modem
UNISOC T820/T770/ T760(UMS9620) arm cpu chip

### ImmortalWrt version

r27736-eb05baff7f

### ImmortalWrt release

SNAPSHOT

### ImmortalWrt target/subtarget

rockchip/armv8

### Device

ZTE F50

### Image kind

Official downloaded image

### Steps to reproduce

Support ZTE F50 5G modem

### Actual behaviour

Support ZTE F50 5G modem

### Expected behaviour

Support ZTE F50 5G modem

### Additional info

Support ZTE F50 5G modem

### Diffconfig

```text
Support ZTE F50 5G modem
```
[ZTE F50 5G](https://forum.openwrt.org/t/support-zte-f50-5g-pocket-wifi-cpe-unisoc-t820-t770-t760-ums9620/218217)

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by libriunk** [comment]
可以考虑一下买个wifi6的手机开热点

**Reply by peterwillcn** [reply]
aha, It's not a matter of money, it's just technical research

**Reply by CypherpunkSamurai** [reply]
@libriunk have you been able to root the device or unlock bootloader? if so its possible. I'm still trying to unlock bootloader with SPRD diag but no success yet.

**Reply by yassinexc10** [reply]
Hello any luck with the zte f50 i am looking to install imortalwrt

**Reply by CypherpunkSamurai** [reply]
bootloader unlock is still not possible. check TomKing062/CVE-2022-38694_unlock_bootloader issues on ZTE F50 i opened one there

**Reply by hypertai** [reply]
https://github.com/Daniel-Hwang/U20-F50
there is a new repo on how to unlock bootloader on ZTE F50

**Comment by CypherpunkSamurai** [comment]
yes. I know. I foundit a month ago. check the issues. we are compiling
custom kernel

On Thu, May 15, 2025 at 7:51 AM hypertai ***@***.***> wrote:

> https://github.com/Daniel-Hwang/U20-F50
> there is a new repo on how to unlock bootloader on ZTE F50
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/immortalwrt/immortalwrt/discussions/1629#discussioncomment-13152323>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP6OSIUUD4CE3WXMPCOOUVD26P233AVCNFSM6AAAAABTTLMKOWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMJVGIZTEMY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***
> com>
>


-- 

*Signed-Off By: Rakesh Chowdhury*
